app/views/shared/_header.html.erb

```diff
@@ -11,44 +11,36 @@
+    <div class="relative mx-4 px-2 lg:mx-0">
+      <span class="font-semibold">
+        Workspace:
+      </span>
+      <%= trunc_account_name(session[:account_name]) %>
+    </div>
   </div>
 
   <div class="flex items-center">
-    <%= link_to t(".log_out"), destroy_user_session_path, method: :delete, data: { turbo_method: :delete } %>
     <button class="flex mx-4 text-gray-600 focus:outline-none">
       <%= svg("bell", class: "w-6 h-6") %>
     </button>
 
-    <div class="relative">
-      <button @click="dropdownOpen = !dropdownOpen" class="relative z-10 block w-8 h-8 overflow-hidden rounded-full shadow focus:outline-none">
+    <div class="relative" data-controller="menu">
+      <button class="relative z-10 block w-8 h-8 overflow-hidden rounded-full shadow focus:outline-none" data-action="click->menu#toggle">
         <img class="object-cover w-full h-full" src="https://images.unsplash.com/photo-1528892952291-009c663ce843?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=296&q=80" alt="Your avatar" />
       </button>
 
-      <div
-        v-show="dropdownOpen"
-        @click="dropdownOpen = false"
-        class="fixed inset-0 z-10 w-full h-full"
-        <%# TODO: remove style line and use js to display profile menu %>
-        style="display: none;"
-        ></div>
-
-      <transition
-        enter-active-class="transition duration-150 ease-out transform"
-        enter-from-class="scale-95 opacity-0"
-        enter-to-class="scale-100 opacity-100"
-        leave-active-class="transition duration-150 ease-in transform"
-        leave-from-class="scale-100 opacity-100"
-        leave-to-class="scale-95 opacity-0">
-        <div
-          v-show="dropdownOpen"
-          class="absolute right-0 z-20 w-48 py-2 mt-2 bg-white rounded-md shadow-xl"
-          style="display:none;"
-          <%# TODO: remove style line and use js to display profile menu %>
+      <div class="fixed inset-0 z-10 w-full h-full hidden" data-menu-target="clearfix"></div>
 
-          >
-          <a href="#" class="block px-4 py-2 text-sm text-gray-700 hover:bg-indigo-600 hover:text-white">Profile</a>
-          <a href="#" class="block px-4 py-2 text-sm text-gray-700 hover:bg-indigo-600 hover:text-white">Products</a>
-          <router-link to="/" class="block px-4 py-2 text-sm text-gray-700 hover:bg-indigo-600 hover:text-white">Log out</router-link>
+      <transition enter-active-class="transition duration-150 ease-out transform" enter-from-class="scale-95 opacity-0" enter-to-class="scale-100 opacity-100" leave-active-class="transition duration-150 ease-in transform" leave-from-class="scale-100 opacity-100" leave-to-class="scale-95 opacity-0">
+        <div class="absolute right-0 z-20 w-48 py-2 mt-2 bg-white rounded-md shadow-xl hidden" data-menu-target="options">
+            <a href="#" class="block px-4 py-2 text-sm text-gray-700 hover:bg-indigo-600 hover:text-white">Profile</a>
+            <% if current_user.is_collaborator? %>
```

Comment:
maybe we should use another dropdown for the accounts or at least add a seperator in this one.